keyboard/src/modules/Keyboard.rb

```diff
@@ -1504,6 +1483,43 @@ def write_udev_rule
     end
   end
 
+  # Checks if the graphical environment is being executed remotely using
+  # "ssh -X"
+  def x11_over_ssh?
+    display = Builtins.getenv("DISPLAY") || ""
+    display.split(":")[1].to_i >= 10
```

Comment:
IMO display.split(":")[1].to_i is a bit more intuitive than display[1..-1].to_i

Using ENV["DISPLAY"] is a good idea: ENV["DISPLAY"].split(":")[1].to_i